packages/wdio-interface/src/index.js

```diff
@@ -1,67 +1,47 @@
-process.title = 'WDIO Testrunner';
+import util from 'util'
+import ansiEscapes from 'ansi-escapes'
 
-import blessed from 'blessed'
-
-import { BOX_STYLE } from './constants'
-
-export class WDIOCLIInterface {
+export default class CLIInterface {
     constructor () {
-        this.screen = blessed.screen({
-            smartCSR: true,
-            log: process.env.HOME + '/blessed-terminal.log',
-            fullUnicode: true,
-            dockBorders: true,
-            ignoreDockContrast: true
-        })
+        this.i = 0
+        this.stdoutBuffer = []
+        this.stderrBuffer = []
+        this.out = process.stdout.write.bind(process.stdout)
```

Comment:
Isn't this where you could use the bind syntax like so? process.stdout.write::process.stdout